person.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+
+  attr_accessor :name, :age
+
+  def of_age
+    @age > 18
+  end
+
+  private :bacon
```

Comment:
the private method should be `of_age` instead of `bacon`  🤣 which doesn't even exist. Kindly fix this error.